canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.jsx

```diff
@@ -178,16 +202,17 @@ class RadiosetControl extends React.Component {
 				row: i
 			};
 			buttons.push(
-				<div key={i} className="properties-radioset-panel">
+				<div key={ControlUtils.getControlId(id, this.uuid)} className="properties-radioset-panel" key={ControlUtils.getControlId(id, this.uuid)}>
```

Comment:
There are 2 keys here and these won't be unique anymore.  You need to include the index I believe.

Reply:
Updated with only one key with index